README.md

```diff
@@ -208,14 +208,16 @@ Options, and their defaults
     limit: 1024 * 1000,
   },
   // disables cluster mode and reloads each component every time it is requested
-  devMode: false,
+  clustering: true,
   // how components will be retrieved,
   getComponent: undefined,
   // configure the logger
   logger: {},
   // the port the app will start on
   port: 8080,
   // whether or not to run in parallel using all available cpus
+  endpoint: '/batch'
```

Comment:
🎉 this is a great (semver-minor) change regardless, so it's no longer hardcoded.